.github/workflows/codeql-build.yml

```diff
@@ -0,0 +1,59 @@
+name: "CodeQL Analysis"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+env:
+  SIMULATION: native
+  ENABLE_UNIT_TESTS: true
+  OMIT_DEPRECATED: true
+
+jobs:
+
+  CodeQL-Build:
+
+    strategy:
+      matrix:
+        buildtype: [debug, release]
```

Comment:
@skliper should we check security stuff for both debug and release builds?

Reply:
release test would be sufficient in my mind... always with OMIT_DEPRECATED